mslice/models/slice/mantid_slice_algorithm.py

```diff
@@ -171,7 +171,7 @@ def compute_symmetrised(self, scattering_data, sample_temp, e_axis, data_rotated
         else:
             rhs = scattering_data[-negative_de_len:, :] * boltzmann_dist[:, None]
             lhs = scattering_data[:-negative_de_len, :]
-            return np.concatenate((rhs, lhs))
+            return np.concatenate((lhs, rhs))
```

Comment:
How did I manage to get those the wrong way around :man_facepalming: :face_with_head_bandage: